### PR TITLE
Raina/create-acct

### PR DESCRIPTION
**Pull request by rainawan**
navigate from sign in to sign up
style create account page

### CODE REVIEW

**Comment by awllms** [comment]
Looks good to me.

**Comment by sonyakim-dev** [comment]
i think it would make more sense to navigate to '/user' after successfully creating an account instead of '/'

File: src/pages/SignUp/UserSignUp.jsx [hunk 1]
```diff
@@ -35,41 +36,52 @@ const UserSignUp = () => {
...
-                placeholder="Enter Your Password"
-                errorMessage={passwordError}
-                isInvalid={passwordError !== ''}
-                onValueChange={() => setPasswordError('')}
-              />
-            </div>
-            <Button color="primary" type="submit" name="submit">
-              Sign Up
-            </Button>
+    <section className="max-w-[60rem]">
```
Comment (sonyakim-dev) [inline src/pages/SignUp/UserSignUp.jsx]: just made mobile responsive and removed redundant styles
```js
    <section>
      <form id="signup-form" onSubmit={handleSignUp}>
        <div className="flex flex-col max-w-[400px] mx-auto gap-4 pt-10">
          <Text h2 className="font-semibold p-5">
            Create Account
          </Text>
          <div className="flex flex-col w-full gap-4">
            <Input
              isRequired
              variant="bordered"
              label="Email"
              name="email"
              type="email"
              size="md"
              placeholder="Enter Your Email"
              errorMessage={emailError}
              isInvalid={emailError !== ''}
              onFocus={() => setEmailError('')}
            />
            <Input
              isRequired
              variant="bordered"
              label="Password"
              name="password"
              type="password"
              size="md"
              placeholder="Enter Your Password"
              errorMessage={passwordError}
              isInvalid={passwordError !== ''}
              onFocus={() => setPasswordError('')}
            />
            <Button color="primary" type="submit" name="submit">
              Sign Up
            </Button>
          </div>
          <p>
            Already have an account?
            <Link
              className="ml-1 font-bold text-md text-gray-300 hover:text-primary hover:underline"
              to="/user">
              Log In
            </Link>
          </p>
        </div>
      </form>
    </section>
```

**Comment by sonyakim-dev** [comment]
lgtm

**Pull request merged**